db/migrate/20160912000001_set_default_run_only_when_new_commit.rb

```diff
@@ -0,0 +1,5 @@
+class SetDefaultRunOnlyWhenNewCommit < ActiveRecord::Migration
+  def change
+    change_column :crons, :run_only_when_new_commit, :boolean, :default => false
```

Comment:
the default would be nil if it's not set, wouldn't it? Or would the default not change in the DB if the old default migration was already run? Is there data already in the table for this column that needs to be changed?